src/Components/SmartComponents/Systems/Systems.js

```diff
@@ -68,12 +69,23 @@ class Systems extends React.Component {
         this.setState({ systemList: newList });
     }
 
+    onRefresh({ page, per_page: perPage }) {
+        const { meta } = this.props.systems;
```

Comment:
If the page_size changes, the page should always reset to 1.
```suggestion
        const { meta } = this.props.systems;
        meta.page_size !== perPage ? (page = 1) : page;
```

Reply:
good catch!